Doc/library/sys.rst

```diff
@@ -905,6 +905,12 @@ always available.
    read, so that you can set this hook there.  The :mod:`site` module
    :ref:`sets this <rlcompleter-config>`.
 
+   .. audit-event:: cpython.run_interactivehook hook sys.__interactivehook__
```

Comment:
I don't understand the audit-event syntax. The event takes one argument (hook), but the audit-event has two arguments. What's going on here?

Reply:
I need to document the syntax somewhere...

Basically, it's:
```
.. audit-event:: event-name list,of,comma,separated,args anchor
```

Where the anchor is used for the backlink from the generated table (which appears to [not be generating right now](https://docs.python.org/3.8/library/audit_events.html)...).